2022-03-29 17:22:41,982 INFO  ==============================================================
2022-03-29 17:22:41,982 INFO  Id:    docker_plugin_finalize
2022-03-29 17:22:41,984 INFO  Dir:   plugin_docker/
2022-03-29 17:22:41,985 INFO  Title: Validate docker plugin finalize command
2022-03-29 17:22:41,985 DEBUG Execution order hint: 0.0
2022-03-29 17:22:41,986 INFO  ==============================================================
2022-03-29 17:22:42,002 DEBUG Process parameters for sudo<docker_images>
  command line : /usr/bin/sudo docker images
    arg #1     : docker
    arg #2     : images
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-03-29 17:22:42,007 DEBUG Waiting up to 600 secs for process sudo<docker_images> (pid 27711)
2022-03-29 17:22:43,568 INFO  Executed sudo<docker_images>, exit status 0
2022-03-29 17:22:43,569 DEBUG Performing regex contains=False grep on file: /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out
2022-03-29 17:22:43,578 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out ... passed
2022-03-29 17:22:43,580 DEBUG Process parameters for sudo<docker_pull>
  command line : /usr/bin/sudo docker pull registry
    arg #1     : docker
    arg #2     : pull
    arg #3     : registry
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_pull.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_pull.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-03-29 17:22:43,585 DEBUG Waiting up to 600 secs for process sudo<docker_pull> (pid 27721)
2022-03-29 17:22:57,281 INFO  Executed sudo<docker_pull>, exit status 0, duration 13 secs
2022-03-29 17:22:57,284 DEBUG Process parameters for sudo<docker_images>
  command line : /usr/bin/sudo docker images
    arg #1     : docker
    arg #2     : images
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-03-29 17:22:57,291 DEBUG Waiting up to 600 secs for process sudo<docker_images> (pid 27832)
2022-03-29 17:22:57,747 INFO  Executed sudo<docker_images>, exit status 0
2022-03-29 17:22:57,748 DEBUG Performing regex contains=True grep on file: /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out
2022-03-29 17:22:57,751 DEBUG Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out successfully matched expression "registry" with line: 'registry            latest              dcaf7118f1b5        5 days ago          19.2MB\n'
2022-03-29 17:22:57,752 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out ... passed
2022-03-29 17:22:57,753 DEBUG --- test execute
2022-03-29 17:22:57,754 DEBUG Process parameters for sudo<plugin_finalize>
  command line : /usr/bin/sudo /etc/tedge/sm-plugins/docker finalize
    arg #1     : /etc/tedge/sm-plugins/docker
    arg #2     : finalize
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/plugin_finalize.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/plugin_finalize.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-03-29 17:22:57,759 DEBUG Waiting up to 600 secs for process sudo<plugin_finalize> (pid 27842)
2022-03-29 17:23:00,474 INFO  Executed sudo<plugin_finalize>, exit status 0
2022-03-29 17:23:00,477 DEBUG Process parameters for sudo<docker_ps_latest>
  command line : /usr/bin/sudo docker ps -q --latest
    arg #1     : docker
    arg #2     : ps
    arg #3     : -q
    arg #4     : --latest
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_ps_latest.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_ps_latest.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-03-29 17:23:00,483 DEBUG Waiting up to 600 secs for process sudo<docker_ps_latest> (pid 27883)
2022-03-29 17:23:00,992 INFO  Executed sudo<docker_ps_latest>, exit status 0
2022-03-29 17:23:00,995 DEBUG --- test validate
2022-03-29 17:23:00,997 DEBUG Process parameters for sudo<docker_images>
  command line : /usr/bin/sudo docker images
    arg #1     : docker
    arg #2     : images
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-03-29 17:23:01,003 DEBUG Waiting up to 600 secs for process sudo<docker_images> (pid 27893)
2022-03-29 17:23:01,458 INFO  Executed sudo<docker_images>, exit status 0
2022-03-29 17:23:01,459 DEBUG Performing regex contains=False grep on file: /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out
2022-03-29 17:23:01,462 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out ... passed
2022-03-29 17:23:01,468 DEBUG --- test cleanup
2022-03-29 17:23:01,469 INFO  
2022-03-29 17:23:01,470 INFO  cleanup:
2022-03-29 17:23:01,470 DEBUG Running registered cleanup function: <bound method DockerPlugin.cleanup_containers of <DockerPluginFinalizeTest object at 0x7609fbf0>>
2022-03-29 17:23:01,471 DEBUG Running registered cleanup function: <bound method DockerPlugin.cleanup_images of <DockerPluginFinalizeTest object at 0x7609fbf0>>
2022-03-29 17:23:01,472 INFO  Removing images scheduled to be cleaned up: ['registry']
2022-03-29 17:23:01,473 DEBUG Process parameters for sudo<containers_for_cleanup>
  command line : /usr/bin/sudo docker rmi -f registry
    arg #1     : docker
    arg #2     : rmi
    arg #3     : -f
    arg #4     : registry
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/containers_for_cleanup.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/containers_for_cleanup.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-03-29 17:23:01,478 DEBUG Waiting up to 600 secs for process sudo<containers_for_cleanup> (pid 27906)
2022-03-29 17:23:01,934 INFO  Executed sudo<containers_for_cleanup>, exit status 0
2022-03-29 17:23:01,935 DEBUG ProcessUser cleanup function done.
2022-03-29 17:23:01,937 INFO  
2022-03-29 17:23:01,937 INFO  Test duration: 19.96 secs
2022-03-29 17:23:01,938 INFO  Test final outcome:  PASSED
2022-03-29 17:23:01,939 INFO  
